packages/block-editor-tools/src/components/post-picker/index.tsx

```diff
@@ -98,6 +90,29 @@ const PostPicker = ({
     setShowModal(false);
   };
 
+  const controls = () => (
+    <ButtonGroup>
+      <Button
+        variant="primary"
```

Comment:
```suggestion
        variant="primary"
```
I feel like with 2 buttons, they shouldn't both be primary. One could be primary and one secondary, or both secondary. Could also experiment with `isDestructive` on the Reset button.

Reply:
Good call! Yeah, that makes sense.